opencog/reasoning/RuleEngine/rules/pln/evaluation-to-member.scm

```diff
@@ -0,0 +1,64 @@
+; ============================================================================= 
+; GeneralEvaluationToMemberRule
+;	EvaluationLink (pred D (ListLink B C))
+;		becomes
+;	MemberLink( B (SatisfyingSetLink( Variable $X 
+;		(EvaluationLink (pred D (ListLink X C))))))
+; -----------------------------------------------------------------------------
+
+(define pln-rule-evaluation-to-member
+	(BindLink
+		(ListLink
+			(VariableNode "$B")
+			(VariableNode "$C")
+			(VariableNode "$D")
+			(TypedVariableLink
+				(VariableNode "$D")
+				(VariableTypeNode "PredicateNode")))
+		(ImplicationLink
+			(EvaluationLink
+				(VariableNode "$D")
```

Comment:
you just want [these line](https://gist.github.com/prateeksaxena2809/bae56405a4aed3898088#file-evaluation-to-member-scm-L34-L36) to be passed to the function, that will actually do more than calculate the TV . The function should check the arguments and produce the possible outputs and set caluclate the TV